users.js

```diff
@@ -398,6 +398,41 @@ function exportUsergroups() {
 }
 importUsergroups();
 
+function cacheGroupData() {
+	var grouplist = Config.grouplist;
+	var groups = Config.groups = {};
```

Comment:
Converting this to <code>Object.create(null)</code> would be a nice improvement.